ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -54,6 +54,26 @@ RSkComponentTextInput::RSkComponentTextInput(const ShadowView &shadowView)
   sem_init(&jsUpdateSem,0,0);
   isKeyRepeateOn=false;
 }
+SkPaint RSkComponentTextInput::convertTextColor ( SharedColor textColor ) {
```

Comment:
RSkDrawUtils already has setColor() , can this be reused instead of defining the same functionality again

Reply:
Agreed & Addressed